examples/laser_STC/analysis_laser_STC.py

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env python3
+
+# Copyright 2025
+#
+# This file is part of HiPACE++.
+#
+# Authors: Xingjian Hui
+# License: BSD-3-Clause-LBNL
+
+import argparse
+import numpy as np
+import scipy.constants as scc
+from lasy.utils.laser_utils import get_dispersion, get_zeta, get_beta
+from lasy.laser import Laser
+from lasy.profiles import FromOpenPMDProfile
+
+lambda0 = .6e-6          # Laser wavelength
```

Comment:
Let's stay close to realistic use cases ok?
```suggestion
lambda0 = .8e-6          # Laser wavelength
```